packages/kuma-gui/src/app/x/components/x-empty-state/XEmptyState.vue

```diff
@@ -79,18 +76,44 @@
               />
             </slot>
           </template>
-        </KEmptyState>
+        </EntityEmptyState>
       </template>
     </template>
   </XI18n>
 </template>
 
 <script lang="ts" setup>
-import { KEmptyState } from '@kong/kongponents'
+import { KUI_COLOR_TEXT_DECORATIVE_AQUA, KUI_ICON_SIZE_50 } from '@kong/design-tokens'
+import { LocationIcon, AnalyticsIcon, MeshIcon } from '@kong/icons'
+import { EntityEmptyState } from '@kong-ui-public/entities-shared'
+import '@kong-ui-public/entities-shared/dist/style.css'
```

Comment:
Could we put this line into here with the others:

https://github.com/kumahq/kuma-gui/blob/e9b0a81d59031989f783a572a1e51a816776d1ad/packages/kuma-gui/src/assets/styles/main.scss#L8-L9